provision/setup.py

```diff
@@ -28,5 +28,6 @@
           'pyopenssl',
           'ipaddr',
           'MarkupSafe<2.0',
+          'boto3',
```

Comment:
Would this need to be packaged in the deb and/or rpm?  If so, we would need to check if and what needs to be added, here:
https://github.com/noironetworks/acc-provision/blob/master/provision/debian/control
and/or here:
https://github.com/noironetworks/acc-provision/blob/master/provision/rpm/acc-provision.spec.in

Reply:
OK, I'll update these